core/templates/dev/head/pages/preferences-page/preferences-page.controller.spec.ts

```diff
@@ -30,7 +31,8 @@ describe('Preferences Controller', function() {
       angular.mock.module('oppia', GLOBALS.TRANSLATOR_PROVIDER_FOR_TESTS));
 
     beforeEach(angular.mock.inject(function(
-        $controller, $http, _$httpBackend_, $rootScope) {
+        _$componentController_, $http, _$httpBackend_, $rootScope) {
+      $componentController = _$componentController_;
```

Comment:
What does the _ at both ends mean?

Reply:
I saw this here - > https://docs.angularjs.org/guide/component#unit-testing-component-controllers